import-export-cli/integration/testdata/sample-api.yaml

```diff
@@ -161,4 +161,7 @@ data: # Contains the meta data of the API
       advertised: false # Whether API is an advertised only API or not
       originalDevPortalUrl: null # The original URL of the Devportal
       apiOwner: admin # The original owner of the API
-      vendor: WSO2 # Vendor of the Advertised only API
\ No newline at end of file
+      vendor: WSO2 # Vendor of the Advertised only API
```

Comment:
As i can remember, this should be in lower case. @Chamindu36 please confirm.

Reply:
This is a different property from what I added for Solace. But this also should be in lower case. After Heshan's feature, this property will be removed.

Reply:
This is a different value from what I added for the solace feature. This was added before for Advertise only API feature and that also should be in lower case as I remember. But after Heshan's feature, this will be changed.